cvmfs/cvmfs_config

```diff
@@ -254,13 +257,14 @@ cvmfs_chksetup() {
     fi
   fi
 
+  AUTOMASTER=$(find /etc -name "auto.master" 2>/dev/null | head -1)
```

Comment:
I've changed the code such that it visits the two presupplied paths and should be easy to extend for more.